### ISSUE TITLE
`rig rstudio` for RStudio Server

### ISSUE THREAD

**Issue by asadow** [r-lib/rig#245]
Is it possible to get `rig rstudio` to work for Rstudio Server? It seemed to "run" but not do anything.

**Comment by gaborcsardi** [comment]
I am not sure how rig could help you with Rstudio Server.

**Comment by kkrick-sdsu** [comment]
I was also looking to see if rig supported RStudio Server. This would be immensely helpful for managing multiple versions of R.

**Comment by gaborcsardi** [comment]
Sure, you can use rig to install multiple R versions on an RStudio Server machine. 

But a single Rstudio Server cannot run multiple R versions at the same time, AFAICT, and I am not sure what rig can do about that.

**Comment by kkrick-sdsu** [comment]
Could rig be used to swap out that one version that RStudio Server is running though? I have been playing around with that approach but have not gotten it to work.

**Comment by gaborcsardi** [comment]
Yes, I _think_ if you leave Rstudio Server settings on the default (i.e. use R from the `PATH`), and you do
```
rig default...
```
that should change the default. It is possible that you need to restart RStudio server, and you might need to even stop it before you change the default, and then start it again after.

But I haven't actually tried this.
